### PR TITLE
Add PlayWright adapter

### PR DESCRIPTION
**Pull request by Dor256**
- Add `PlayWright` adapter
- Remove `Clear` key from key definitions

### CODE REVIEW

File: adapters/playwright/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+    const isElementInViewport = (selector: string) => {
+        const rect = document.querySelector(selector)?.getBoundingClientRect();
+        if (!rect) {
+            return false;
+        }
+        return (
+            rect.top >= 0 &&
+            rect.left >= 0 &&
+            rect.bottom <= (window.innerHeight || document.documentElement.clientHeight) &&
+            rect.right <= (window.innerWidth || document.documentElement.clientWidth)
```
Comment (Dor256) [inline adapters/playwright/src/index.ts:120]:
```suggestion
            rect.bottom <= (window.innerHeight ?? document.documentElement.clientHeight) &&
            rect.right <= (window.innerWidth ?? document.documentElement.clientWidth)
```

File: adapters/playwright/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+	"name": "@unidriver/playwright",
+	"version": "1.2.2",
```
Comment (matan04) [inline adapters/playwright/package.json:3]:
```suggestion
	"version": "1.0.0",
```

**Review by matan04 — APPROVED** [review]
Amazing ✨

**Review by GabiGrin — APPROVED** [review]
⭐ ⭐ ⭐ ⭐ ⭐

**Pull request merged**